lib/hdkey.js

```diff
@@ -33,10 +34,10 @@ Object.defineProperty(HDKey.prototype, 'privateKey', {
   },
   set: function (value) {
     assert.equal(value.length, 32, 'Private key must be 32 bytes.')
-    assert(secp256k1.privateKeyVerify(value) === true, 'Invalid private key')
+    assert(secp256k1.isPrivate(value) === true, 'Invalid private key')
 
     this._privateKey = value
-    this._publicKey = Buffer.from(secp256k1.publicKeyCreate(value, true))
+    this._publicKey = Buffer.from(necc.getPublicKey(value, true))
```

Comment:
`@bitcoinerlab/secp256k1` has this function under another name.

You shouldn't need to import noble directly.

Reply:
`pointFromScalar` is the name of the function.

Reply:
ah nice, thanks!